bottlecache/src/main.rs

```diff
@@ -13,7 +13,7 @@ use common::TestsuiteResult;
 #[derive(StructOpt, Debug)]
 pub struct Args {
     #[structopt(short, long, help = "Personal access token if available")]
```

Comment:
```suggestion
    #[structopt(short, long, help = "Personal access token")]
```

not a change you made in this PR, but I think it would be nice so that the token does not look "optional" when running `--help`

Reply:
Done!